cli/cli.ts

```diff
@@ -4816,7 +4817,8 @@ function internalGenDocsAsync(docs: boolean, locs: boolean, fileFilter?: string,
         docs,
         locs,
         fileFilter,
-        createOnly
+        createOnly,
+        ignoreTests: true
```

Comment:
Note that this is inside the "internalGenDocsAsync" function which seemingly has nothing to do with API caching. However due to the fact that we keep around api info between compiler calls, the info we build here ends up affecting what goes into our cache.